requirements-dev.txt

```
@@ -11,7 +11,7 @@ pytest
 ### coverage
 coverage
 pytest-cov
-black==21.5b2
+black
```

```suggestion
black==22.3.0
```

I'd suggest to be explicit and pin the version strictly.